exporter/datadogexporter/metrics_translator.go

```diff
@@ -205,10 +207,22 @@ func mapSummaryMetrics(name string, slice pdata.SummaryDataPointSlice, quantiles
 		tags := getTags(p.LabelsMap())
 		tags = append(tags, attrTags...)
 
-		ms = append(ms,
-			metrics.NewGauge(fmt.Sprintf("%s.count", name), ts, float64(p.Count()), tags),
-			metrics.NewGauge(fmt.Sprintf("%s.sum", name), ts, p.Sum(), tags),
-		)
+		// count and sum are increasing; we treat them as cumulative monotonic sums.
+		{
+			countName := fmt.Sprintf("%s.count", name)
+			countKey := metricDimensionsToMapKey(countName, tags)
+			if dx, ok := putAndGetDiff(prevPts, countKey, ts, float64(p.Count())); ok {
+				ms = append(ms, metrics.NewCount(countName, ts, dx, tags))
+			}
+		}
+
+		{
+			sumName := fmt.Sprintf("%s.sum", name)
+			sumKey := metricDimensionsToMapKey(sumName, tags)
+			if dx, ok := putAndGetDiff(prevPts, sumKey, ts, p.Sum()); ok {
```

Comment:
We assume here that sums are increasing as we do in the OpenMetrics check. See this protobuf comment for details https://github.com/open-telemetry/opentelemetry-proto/blob/8672494217bfc858e2a82a4e8c623d4a5530473a/opentelemetry/proto/metrics/v1/metrics.proto#L611-L615